README.md

```diff
@@ -1081,6 +1084,16 @@ security:
 
 Confused? Read [Securing Gatus with OIDC using Auth0](https://twin.sh/articles/56/securing-gatus-with-oidc-using-auth0).
 
+### TLS Encryption
+Gatus supports basic encryption with TLS. To enable this, certificate files in PEM format have to be provided.
+The example below shows an example configuration which makes gatus respond on port 4443 to HTTPS requests.
+
+```yaml
+web:
+  port: 4443
+  certificate-file: "server.crt"
+  private-key-file: "server.key"
```

Comment:
Call me nitpicky, but I think I would prefer if this was under `tls`

Something like
```yaml
web:
  port: 443
  tls:
    certificate-file: "server.crt"
    private-key-file: "server.key"
```

Reply:
Yes, looks better. I've done that.